### PR TITLE
[web3torrent] Fix typo in drift env var

### LINKED ISSUE #2043: [web3torrent] No drift button
I've not seen this appearing recently.

https://github.com/statechannels/monorepo/blob/921a185ebfe72259e71fd29835a51a70f3251e57/packages/web3torrent/src/index.tsx#L22

### PR DESCRIPTION
**Pull request by geoknee**
Closes #2043

### CODE REVIEW

File: packages/web3torrent/config/webpack.config.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ module.exports = function(webpackEnv) {
         [
           'VERSION',
           'CHAIN_NETWORK_ID',
-          'DRIFT_CHATBOX_ID',
+          'DRIFT_CHATBOX_APP_ID',
```
Comment (geoknee) [inline packages/web3torrent/config/webpack.config.js:78]: Do we still want to whitelist environment variables? As is we need to add a new env var in two places (`.env.something` as well as here). Easy to miss this, and it doesn't match the pattern used in our other apps.

Comment (snario) [reply packages/web3torrent/config/webpack.config.js:78]: I don't think it's that useful to be whitelisting them

Comment (geoknee) [reply packages/web3torrent/config/webpack.config.js:78]: https://github.com/statechannels/monorepo/pull/2045

**Comment by geoknee** [comment]
In favour of #2045

**Pull request closed without merging**